### PR TITLE
PE-3990: (fine tune): fix sizes and fonts for the estimation view

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/2qplrgcj5dprg
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/1tv7lsg7e8qp0

### CODE REVIEW

**Review by karlprieb — APPROVED** [review]

File: lib/components/top_up_dialog.dart [hunk 1]
```diff
@@ -361,32 +361,29 @@ class _PresetAmountSelectorState extends State<PresetAmountSelector> {
           const SizedBox(height: 8),
           Text(
             // TODO: Localize
-            'ArDrive Credits will be automatically applied to your wallet, and you can start using them right away.',
+            'ArDrive Credits will be automatically added to your Turbo balance, and you can start using them right away.',
```
Comment (matibat) [inline lib/components/top_up_dialog.dart:364]: TODO - @matibat 
Wordings changed

File: lib/components/turbo_balance_widget.dart [hunk 2]
```diff
@@ -112,7 +112,7 @@ class _TurboBalanceState extends State<TurboBalance> {
                   crossAxisAlignment: CrossAxisAlignment.center,
                   children: [
                     Text(
-                      '${winstonToAr(balance).toStringAsFixed(5)} ${appLocalizationsOf(context).creditsTurbo}',
+                      '${convertCreditsToLiteralString(balance)} ${appLocalizationsOf(context).creditsTurbo}',
```
Comment (matibat) [inline lib/components/turbo_balance_widget.dart:115]: There's a difference in the decimal digits. Here it's 5, in the method it's 4.

File: pubspec.lock [hunk 3]
```diff
@@ -88,7 +88,7 @@ packages:
     description:
       path: "."
       ref: PE-3693-implementing-balance-and-price-estimate-view-for-turbo-top-up-only-usd
-      resolved-ref: bc704b49c498553d09354bd857a6127419b8ad2f
+      resolved-ref: ccf36956c1e9d9d132e9af4e7e29f36c06cbc857
```
Comment (matibat) [inline pubspec.lock:91]: We have to use a released version

**Review by matibat — COMMENTED** [review]
Left some comments. Also, can we have some screenshots?

File: pubspec.lock (hunk 3 above)
Comment (karlprieb) [reply pubspec.lock:91]: It will eventually. Currently this PR is targetting another PR

**Pull request merged**